### DISCUSSION TITLE
Video/Screenshot setup example

### DISCUSSION THREAD

**Discussion by marchingon12** [DirectoryLister/DirectoryLister#615] [Ideas & Features]
If someone could write a guide with a video tutorial using nginx and PHP would be awesome, as we really don't know how to set it up even after reading through the docs multiple times...

**Comment by PHLAK** [comment]
There's nothing special about installing Directory Lister. All you need is a web server (NGINX in your case) set up to server PHP files, then [extract and copy the files as stated in the documentation](https://docs.directorylister.com/getting-started/installation).

If you need help setting up a web server there are plenty of tutorials online. For example, [this is a pretty good](https://www.digitalocean.com/community/tutorials/how-to-install-linux-nginx-mysql-php-lemp-stack-on-ubuntu-20-04) one.
